### ISSUE TITLE
Nuget Package Manager for single project fails to load after adding RingCentralSDK

### ISSUE THREAD

**Issue by Thierry-S** [ringcentral/ringcentral-csharp#47]
Posted this on Stack Overflow, then RingCentral devcommunity, and now I've been told to put it here.

I'm on VS 2015, targeting .net 4.5.2, my "Nuget Package Manager for Visual Studio" in Extensions and Updates shows 3.4.4.1321.

I added all dependencies of RingCentralSDK package separately:

    - Microsoft.Bcl
    - Microsoft.Bcl.Async
    - Microsoft.Bcl.Build
    - Microsoft.Net.Http
    - Newtonsoft.Json
    - Portable.BouncyCastle-Signed
    - PubnubPCL

Then I right click on my project, pick "Manage Nuget Packages...", the tab opens, in the "Installed" subtab I can see the 7 packages.

Now, I install the RingCentralSDK package. My output is fine:

    Attempting to gather dependency information for package 'RingCentralSDK.1.0.0' with respect to project 'MyProj', targeting '.NETFramework,Version=v4.5.2'
    Attempting to resolve dependencies for package 'RingCentralSDK.1.0.0' with DependencyBehavior 'Lowest'
    Resolving actions to install package 'RingCentralSDK.1.0.0'
    Resolved actions to install package 'RingCentralSDK.1.0.0'
    Adding package 'RingCentralSDK.1.0.0' to folder 'C:\code\packages'
    Added package 'RingCentralSDK.1.0.0' to folder 'C:\code\packages'
    Added package 'RingCentralSDK.1.0.0' to 'packages.config'
    Successfully installed 'RingCentralSDK 1.0.0' to MyProj
    ========== Finished ==========

Now, I check again the Manage Nuget Packages for this specific project, and instead of seeing the list of packages, I get a yellow band:

    Error occured                  Show erros in output         Dismiss

And in the "Package Manager" output, the error is:

    ZIP archive contains unsupported data structures.

In my packages folder, the folder "RingCentralSDK.1.0.0" seems ok, but then I don't anything about nuget packages format.

I cannot uninstall RingCentral via the package manager as it's not showing, nor any other one, which is really annoying. I had an error when trying to do it via the command line. I reverted my changes (or remove the line in packages.config for my project), and the Nuget Package Manager tab works fine again. This is also affecting the "Managet Nuget Packages for Solution".

What can I do to fix this?

**Comment by tylerlong** [comment]
Let me investigate and get back to you later. One possibility: when we created the NuGet package, we used the latest NuGet binary `NuGet Version: 3.4.3.855` on Mac.  Maybe there are compatibility issues between NuGet binary and the NuGet Package Manager.

**Comment by tylerlong** [comment]
I just did a quick test by creating a brand new project with Visual Studio 2015 on Windows 10.

Visual Studio Menu :point_right: Tools :point_right: NuGet Package Manager :point_right: Package Manager Console

In Package Manager Console, I executed the following command:

```
Install-Package RingCentralSDK
```

Everything seems find and here is the final `packages.config`:

```xml
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net40" />
  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net40" />
  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net40" />
  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net40" />
  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net40" />
  <package id="Portable.BouncyCastle-Signed" version="1.7.0.2" targetFramework="net40" />
  <package id="PubnubPCL" version="3.7.5.0" targetFramework="net40" />
  <package id="RingCentralSDK" version="1.0.0" targetFramework="net40" />
</packages>
```

The experiment described above proves that at  least it works with the Package Manager Console. 

It seems that you are using some graphical tool. I have never used that before. Let me have a try.

**Comment by tylerlong** [comment]
I can reproduce the issue with the GUI tool. I am investigating.

**Comment by tylerlong** [comment]
Fixed in `RingCentralSDK 1.0.1-alpha1`

It turns out that the NuGet package created on Mac is not compatible with Visual Studio NuGet GUI tool.

You can install the latest version by 

```
Install-Package RingCentralSDK -Pre
```

Or if you prefer GUI:

![image](https://cloud.githubusercontent.com/assets/733544/16066923/157dfe8c-32ea-11e6-8895-fb12dbcb654f.png)

**Comment by Thierry-S** [comment]
At first I thought I still had the error after installing 1.0.1alpha. So I removed the reference, closed VS, deleted both 1.0.0 and 1.0.1alpha from the packages folder, re-opened VS, readded the reference.... and now it's fine. Thank you.

![image](https://cloud.githubusercontent.com/assets/13853345/16073483/f5ae994c-32dd-11e6-9f46-71a99fc9370e.png)

**Comment by tylerlong** [comment]
Today I did some testing and I came to the following conclusion:

1. New project will have no problem with 1.0.1 version or later
1. For old project which have 1.0.0 installed. You need to follow the following steps:
    1. UnInstall RingCentralSDK
    1. Close VS
    1. Delete packages folder in your project
    1. Delete packages folder in ~/.nuget/
    1. Launch VS
    1. Install RingCentralSDK

I need to do the above because nuget has cache. If I don't do it the problem just persists. I don't know the exactly reason but the above steps did work for me.
